actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors instead of ignoring them to make diagnosing socket errors easier
+        logger.error "Error occurred while trying to write to the Action Cable socket: #{message}"
```

Comment:
"instead" in the comment sounds fixated on the past: that explanation belongs in a commit message (why we changed this), not in the code (why this is as it is).

As we're revisiting, let's tighten up the message, to give the "real" message more room.. how about `"Error writing to Action Cable socket: #{message}"`?